schemas/cloud-gateway-schema.json

```
@@ -32,6 +32,80 @@
                                                 }
                                             }
                                         },
+                                        "security" :{
+                                            "type": "object",
+                                            "description": "Security settings",
+                                            "properties": {
+                                                "oauth2": {
+                                                    "type": "object",
+                                                    "description": "Enable OAuth2 login flow",
+                                                    "properties": {
+                                                        "client": {
+                                                            "type": "object",
+                                                            "description": "OAuth2 client settings",
+                                                            "properties": {
+                                                                "registration": {
+                                                                    "type": "object",
+                                                                    "description": "OAuth2 client registration settings",
+                                                                    "patternProperties": {
+                                                                        "$*.^": {
```

Can we specify the name more strictly?
```suggestion
                                                                        "$[a-Z0-9_]+^": {
```